### DISCUSSION TITLE
Document how to run the addon in development mode

### DISCUSSION THREAD

**Discussion by jeremyvillalobos** [hlorus/CAD_Sketcher#376] [Feature Requests]
Hello:

I am new to blender and to CAD Sketcher.  I have done a few small 3D models in blender but needed the measurement helpers since I am starting to play with 3D printing.   But the addon does not behave very well on my Ubuntu 20.04 AMD setup with Radeon video card.  

I see a bunch of python stack traces.    This is with CAD Sketcher 0.27.1  and blender 3.5.0 

I am a Software Engineer and have been writing python for about 7 years and I am here to help.  But first I need some help to start helping. 

I would like some documentation on how I would run the addon in development mode.  That ways I can troubleshoot the errors as I see them, them push fixes.

Thank you.

**Comment by hlorus** [comment]
Hey there, great to here that you'd like to help. There's currently no development/release mode. For development i'd suggest to install th addon with the git method decribed [here](https://hlorus.github.io/CAD_Sketcher/installation/).
If you're using vscode there's already a configuration in the addonfolder so you'll just need to install debugpy for debugging. Otherwise make sure that you'll have access to the console output of blender.
Inside of the addons preferences you can set the debug level which might be helpful. There's also a debug mode there which is used to hide some additional settings and features.

**Comment by hlorus** [comment]
The [contribute channel](https://discord.gg/pqFWkwJy) is a great place if you'd like help.
If you'd like to expand on the documentation this is as simple as editing the files in the "docs" subfolder.
